Lib/gzip.py

```diff
@@ -95,9 +95,11 @@ def __init__(self, filename=None, mode=None,
         if filename is None:
             # Issue #13781: os.fdopen() creates a fileobj with a bogus name
             # attribute. Avoid saving this in the gzip header's filename field.
-            if hasattr(fileobj, 'name') and fileobj.name != '<fdopen>':
-                filename = fileobj.name
-            else:
+            filename = getattr(fileobj, 'name', '')
+            if (
```

Comment:
This formatting looks uncommon. I suggest to remove suspicious newlines. Could it fit in one line?

Reply:
Even with `basestring`, it's one char too long to fit on a line. This format is in PEP8 (and flake8 and company are fine with it). I prefer it to this:

```python
if (not isinstance(filename, basestring) or
    filename == '<fdopen>'):
    filename = ''
```

because it is hard to see from the next line. If you'd rather some other style that's fine, but there doesn't seem to be a precedent in this file?

Reply:
I can write it in the line of length 79. This is in the [PEP 8 limit](https://www.python.org/dev/peps/pep-0008/#maximum-line-length). Even if it be out of the limit, [PEP 8 suggests the following options](https://www.python.org/dev/peps/pep-0008/#indentation):

* No extra indentation.
```python
            if (not isinstance(filename, basestring) or
                filename == '<fdopen>'):
                filename = ''
```
* Add a comment, which will provide some distinction in editors supporting syntax highlighting.
```python
            if (not isinstance(filename, basestring) or
                filename == '<fdopen>'):
                # comment
                filename = ''
```
* Add some extra indentation on the conditional continuation line.
```python
            if (not isinstance(filename, basestring)
                    or filename == '<fdopen>'):
                filename = ''
```

Reply:
Yes, I'm familiar with the PEP. I got rid of the parens and put everything on one line.